.drone.yml

```diff
@@ -298,7 +298,7 @@ steps:
     image: codedotorg/cdo-ci:1.18
     pull: always
     environment:
-      CI_JOB: 'ui_tests'
+      CI_JOB: 'prepare_ci_tests'
```

Comment:
We're naming this `prepare_ci_tests` but it's actually only preparing for "UI" tests, right?